modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/execution/SdkInternalContext.java

```diff
@@ -199,5 +201,13 @@ public CoreEvent getOperationEvent() {
     public ExecutorCallback getCallback() {
       return callback;
     }
+
+    public void setExecutionContextAdapter(ExecutionContextAdapter executionContextAdapter) {
```

Comment:
should this actually be mutable? @elrodro83 what say you?

Reply:
no it shouldn't there are other places already in use around here where this state may be kept (sdk context, for instance)